include/simpleble/Peripheral.h

```diff
@@ -30,12 +30,13 @@ class Peripheral {
     std::vector<BluetoothService> services();
     std::map<uint16_t, ByteArray> manufacturer_data();
 
-    ByteArray read(BluetoothUUID service, BluetoothUUID characteristic);
-    void write_request(BluetoothUUID service, BluetoothUUID characteristic, ByteArray data);
-    void write_command(BluetoothUUID service, BluetoothUUID characteristic, ByteArray data);
-    void notify(BluetoothUUID service, BluetoothUUID characteristic, std::function<void(ByteArray payload)> callback);
-    void indicate(BluetoothUUID service, BluetoothUUID characteristic, std::function<void(ByteArray payload)> callback);
-    void unsubscribe(BluetoothUUID service, BluetoothUUID characteristic);
+    // TODO: The ByteArrays in the following functions could probably be passed by reference as well
```

Comment:
I agree, we can add it to this PR.

Reply:
I'll do it, I left the TODO there because I was going to check what kind of type `ByteArray` actually is, but I forgot about it.